src/server/metrics.rs

```diff
@@ -23,6 +24,7 @@ pub struct MetricTimer {
 #[derive(Debug, Clone)]
 pub struct Metrics {
     client: Option<StatsdClient>,
+    tags: Option<Tags>,
```

Comment:
FYI HashMap::new doesn't allocate, so the Option dance doesn't seem worth it

Reply:
Ah, actually, thank you. I need to make sure that I consistently use type `Tags` here.